programs/relayer/Cargo.toml

```diff
@@ -21,7 +21,10 @@ serde      = { workspace = true, default-features = true }
 serde_json = { workspace = true, default-features = true }
 prost      = { workspace = true, default-features = true }
 
-alloy = { workspace = true, features = ["full", "node-bindings"] }
+alloy-primitives = { workspace = true }
+alloy-provider   = { workspace = true, features = ["reqwest"] }
+alloy-network    = { workspace = true }
+alloy-transport  = { workspace = true }
```

Comment:
This is not needed since this compiles to an executable anyway (no on chain logic).